tests/mq_llm_engine/test_error_handling.py

```diff
@@ -153,27 +153,20 @@ async def test_failed_abort(tmp_socket):
         await client.check_health()
 
         # Trigger an abort on the client side.
-        async def bad_abort_after_2s():
-            await asyncio.sleep(2.0)
-            await client.abort(request_id="foo")
+        # This request ID does not exist, and will cause the engine to error
```

Comment:
This test would always hang on my dev box- it looked like the generation finished before the bad abort, causing a failure, and the engine was not able to clean up properly for the test to finish.

This still tests that the `KeyError` is raised and stored so that a future request fails with it

Reply:
Do you mean that calling abort with a request id that's already finished, it will raise an error (and/or even crash the server?)  ... that's something else we should fix I think if that's the case...

Reply:
yeah, I commented about it in the original PR. Didn't want to change functionality here but I think I agree we can let the engine live in this case

Reply:
Ah got it thanks @robertgshaw2-neuralmagic.